src/components/InMarketAsset/InMarket.tsx

```diff
@@ -66,6 +59,7 @@ const InMarket = () => {
      (history.push('/inventory/home/draft')
     )
 
+  const expired = auction.expireTimestamp - Date.now()
```

Comment:
to resolve auction expiretime is null should be validate that auction data is called